### ISSUE TITLE
Accessibility validation is incorrect when switch image's purpose

### ISSUE THREAD

**Issue by ecureuill** [solex2006/SELIProject#178]
## Expected Behavior

When user changes the Image's purpose option, system should keep the validation of the field that is still visible.

## Current Behavior

Accessibility validation is ignoring field already filled up.

## Steps to Reproduce

In Accessibility Settings form of a Image
1. Fill up the _Short Description field_ 
1. Check **Image of Text** option
    1. Notice that accessibility status shows as 0/1 (should be 1/1)
1. Check **Complex Images**
    1. Notice that accessibility status shows as 0/2 (should be 0/2)
1. Fill up _Long Description_ field
    1. Notice that accessibility status shows as 1/2 (should be 1/2)
1. Input any character in _Short Description_ field 
    1. Notice that accessibility status shows as 2/2 (correct) 
1. Check **Image of Text** option
    1. Notice that accessibility status shows as 0/1 (should be 1/1)
1. Check **Complex Images**
    1. Notice that accessibility status shows as 0/2 (should be 0/2)

![image](https://user-images.githubusercontent.com/993369/90984469-841ca880-e54b-11ea-9665-80519a11cc0a.png)

![image](https://user-images.githubusercontent.com/993369/90984666-d7432b00-e54c-11ea-8943-89b6546028de.png)

## Environment

https://vm2161.kaj.pouta.csc.fi/

## Possible Solution

Call validation function when radiobutton value changes 

## Related issues
 

This interfere in:
Accessibility Report feature #45 
- Will present wrong data
- Could prevent teacher from publish course: if teacher selects a inclusion goal that depends on Images accessibility and validations is wrongly set as misconfigured, thus Inclusion Goal will be report as not accomplished and Publish button will not be enabled (#45)
- If course's inclusion goal does not rely on Image's accessibility, the course will be eventually published, but students will get wrong data about accessibility
